DigitalLearningSolutions.Data.Migrations/202203161448_RestructureAdminAndDelegateTables.cs

```diff
@@ -53,23 +60,41 @@ public override void Up()
             Delete.Index("IX_Candidates_CentreID_LastName").OnTable("DelegateAccounts");
             Delete.Index("IX_Candidates_CentreID_FirstName_LastName").OnTable("DelegateAccounts");
             Delete.Index("IX_Candidates_Active_CentreID").OnTable("DelegateAccounts");
+            Delete.Index("IX_Candidates_Active_CentreID_LastName ").OnTable("DelegateAccounts");
+            Delete.Index("IX_Candidates_CentreID_DateRegistered").OnTable("DelegateAccounts");
             Delete.Index("IX_Candidates_CentreID_FirstName").OnTable("DelegateAccounts");
             Delete.Index("IX_Candidates_CentreID").OnTable("DelegateAccounts");
+            Delete.Index("IX_Candidates_CentreIDAliasID").OnTable("DelegateAccounts");
+            Delete.Index("IX_Candidates_CentreID_EmailAddress").OnTable("DelegateAccounts");
+
             Alter.Column("CandidateNumber").OnTable("DelegateAccounts").AsString(20).NotNullable();
+
             Create.Index("IX_DelegateAccounts_CandidateNumber").OnTable("DelegateAccounts").OnColumn("CandidateNumber")
                 .Ascending().WithOptions().Unique().WithOptions().NonClustered();
             Create.Index("IX_DelegateAccounts_Active_CentreID").OnTable("DelegateAccounts").OnColumn("Active")
                 .Ascending()
                 .OnColumn("CentreID").Ascending().WithOptions().NonClustered();
+            Create.Index("IX_DelegateAccounts_CentreID_DateRegistered").OnTable("DelegateAccounts").OnColumn("CentreID")
+                .Ascending().WithOptions().NonClustered();
             Create.Index("IX_DelegateAccounts_CentreID").OnTable("DelegateAccounts").OnColumn("CentreID").Ascending()
                 .WithOptions().NonClustered();
+            Create.Index("IX_DelegateAccounts_CentreID_EmailAddress").OnTable("DelegateAccounts").OnColumn("CentreID")
+                .Ascending()
+                .OnColumn("EmailAddress").Ascending().WithOptions().NonClustered();
 
             Rename.Column("EmailAddress").OnTable("DelegateAccounts").To("Email");
             Alter.Table("DelegateAccounts").AddColumn("EmailVerified").AsDateTime().Nullable();
             Alter.Column("Answer1").OnTable("DelegateAccounts").AsString(100).Nullable();
             Alter.Column("Answer2").OnTable("DelegateAccounts").AsString(100).Nullable();
             Alter.Column("Answer3").OnTable("DelegateAccounts").AsString(100).Nullable();
 
+            Delete.ForeignKey("FK_Candidates_Centres").OnTable("DelegateAccounts");
+            Create.ForeignKey("FK_DelegateAccounts_Centres").FromTable("DelegateAccounts").ForeignColumn("CentreID")
+                .ToTable("Centres").PrimaryColumn("CentreID");
+            Delete.ForeignKey("FK_Candidates_JobGroups").OnTable("DelegateAccounts");
+            Create.ForeignKey("FK_DelegateAccounts_JobGroups").FromTable("DelegateAccounts").ForeignColumn("JobGroupID")
+                .ToTable("JobGroups").PrimaryColumn("JobGroupID");

```

Comment:
JobGroup is going in the new Users table, so I don't think this new creation is necessary.

Reply:
Thanks for reminding me, I've removed the recreation